src/orchard/java.clj

```diff
@@ -277,8 +277,9 @@
           source-file-url (src-files/class->source-file-url c)
           result (misc/deep-merge (reflect-info (reflection-for c))
                                   (when source-file-url
-                                    {:file relative-source-path
-                                     :file-url source-file-url})
+                                    {:file     source-file-url
```

Comment:
You might want to add a note here that both keys are essentially the same for historical reason.s

Reply:
Done